slovak/marsmelaci.md

```diff
@@ -1,2 +1,6 @@
-Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z marshmallow na to, že máš v nose kyslú rybičku.
-
+Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z marshmallow, až na to, že máš v nose kyslú rybičku.
+Štípe ťa tam. Čo skúsiš? 
+[Nič.](nic/sexi-kocka.md)
+[Skúsiš ju vytiahnuť.](Vytiahnut/tahanie.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.